.github/workflows/performance.yml

```
@@ -0,0 +1,160 @@
+concurrency:
+  # The concurrency group contains the workflow name and the branch name for pull requests
+  # or the commit hash for any other events.
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+env:
+  LOCAL_DIR: build
+
+jobs:
+  # Runs the performance test suite.
+  #
+  # Performs the following steps:
+  # - Sets environment variables.
+  # - Checks out the repository.
+  # - Sets up Node.js.
+  # - Log debug information.
+  # - Installs npm dependencies.
+  # - Builds WordPress to run from the `build` directory.
+  # - Starts the WordPress Docker container.
+  # - Logs the running Docker containers.
+  # - Logs Docker debug info.
+  # - Installs WordPress.
+  # - Installs the Server Timing MU plugin.
+  # - Install wordpress importer plugin and import dummy data.
+  # - Update permalink structure.
+  # - Run the Performance tests (current commit).
+  # - Print Performance tests results.
+  # - Update the environment to 6.1.1
+  # - Run Performance tests (6.1.1)
+  # - Print Performance tests results (6.1.1).
+  # - Set commit details.
+  # - Publish performance results.
+  # - Ensures version-controlled files are not modified or deleted.
+  performance:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'WordPress/wordpress-develop' || github.event_name == 'pull_request' }}
+
+    steps:
+      - name: Configure environment variables
+        run: |
+          echo "PHP_FPM_UID=$(id -u)" >> $GITHUB_ENV
+          echo "PHP_FPM_GID=$(id -g)" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+
+      - name: Set up Node.js
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Log debug information
+        run: |
+          npm --version
+          node --version
+          curl --version
+          git --version
+          svn --version
+          locale -a
+
+      - name: Install npm Dependencies
+        run: npm ci
+
+      - name: Build WordPress
+        run: npm run build
+
+      - name: Start Docker environment
+        run: |
+          npm run env:start
+
+      - name: Log running Docker containers
+        run: docker ps -a
+
+      - name: Docker debug information
+        run: |
+          docker -v
+          docker-compose -v
+          docker-compose run --rm mysql mysql --version
+          docker-compose run --rm php php --version
+          docker-compose run --rm php php -m
+          docker-compose run --rm php php -i
+          docker-compose run --rm php locale -a
+
+      - name: Install WordPress
+        run: npm run env:install
+
+      - name: Install WordPress Importer plugin and import mock data
+        run: |
+          npm run env:cli -- plugin install wordpress-importer --activate
```

The `wordpress-importer` is [already installed as part of the install script](https://github.com/WordPress/wordpress-develop/blob/trunk/tools/local-env/scripts/install.js#L23). So I think we just need to activate it here?

Also, the other WP CLI commands needed `--path=/var/www/build` to run without issue. Do we need the same for this script? I'm curious why only _some_ of the commands need it.
```suggestion
          npm run env:cli -- plugin activate wordpress-importer
```

---

Thanks for the review. If we remove the installation from command then it show `The 'wordpress-importer' plugin could not be found.` error in GH workflow.

Will add `--path=/var/www/build` for scripts. cc. @joemcgill

---

Sorry, I was mistaken in this. The install script does install the plugin, but not in the normal `wp-content` folder. Instead it goes to the `tests/phpunit/data/plugins/wordpress-importer` directory, which makes sense why it wouldn't be present for activation.

We could leave this as is, or we could instead run `cp tests/phpunit/data/plugins/wordpress-importer build/wp-content/plugins` to avoid having to deal with network connections in this step.